doc/lua_api.md

```diff
@@ -6839,6 +6839,7 @@ You can find mod channels communication scheme in `doc/mod_channels.png`.
     * Server joins channel `channel_name`, and creates it if necessary. You
       should listen for incoming messages with
       `core.register_on_modchannel_message`
+    * This returns a ModChannelRef object. You can learn how to use this in the [ModChannel](https://github.com/luanti-org/luanti/blob/master/doc/lua_api.md#modchannel) section.
```

Comment:
`ModChannelRef` does not exist. Shortening it a bit, trimming to 90 characters max.

```suggestion
    * Returns a `ModChannel` object. You can learn how to use this in the
      [ModChannel](#modchannel) section.
```